extern/boostd-data/client/client.go

```diff
@@ -123,3 +124,33 @@ func (s *Store) GetOffset(pieceCid cid.Cid, hash mh.Multihash) (uint64, error) {
 
 	return resp, nil
 }
+
+func (s *Store) RemoveDeal(pieceCid cid.Cid, dealId uuid.UUID) error {
```

Comment:
@LexLuthr please keep names on the Client and names on the RPC server consistent, otherwise it get confusing quickly.

removeDealForPiece => removeDeal
removeAllMultihashes => removeMultihashes

etc.

Reply:
Fixed. I have made a couple of more changes to tweak record removal logic.